src/styles/themes/default.ts

```diff
@@ -81,16 +82,14 @@ const darkTheme = {
     skeletonLHNIn: colors.darkBorders,
     skeletonLHNOut: colors.darkDefaultButton,
     QRLogo: colors.green400,
-    starDefaultBG: 'rgb(254, 228, 94)',
-};
+} satisfies ThemeColorsWithoutPageBackgroundColors;
 
-darkTheme.PAGE_BACKGROUND_COLORS = {
-    [SCREENS.HOME]: darkTheme.sidebar,
-    [SCREENS.SETTINGS.PREFERENCES]: colors.blue500,
-    [SCREENS.SETTINGS.WORKSPACES]: colors.pink800,
-    [ROUTES.SETTINGS_STATUS]: colors.green700,
-    [ROUTES.I_KNOW_A_TEACHER]: colors.tangerine800,
-    [ROUTES.SETTINGS_SECURITY]: colors.ice500,
-};
+const darkTheme = {
+    ...darkThemeWithoutPageBackgroundColors,
+    PAGE_BACKGROUND_COLORS: {
+        [SCREENS.HOME]: darkThemeWithoutPageBackgroundColors.sidebar,
```

Comment:
we should only reference colors. here and not the theme.
```suggestion
        [SCREENS.HOME]: colors.darkHighlightBackground,
```

Reply:
I agree, i mainly implemented it this way because we still used theme colors inside the theme after the [light theme PR](https://github.com/Expensify/App/pull/23158) was merged. Gonna change this and remove the spread